docs/hugo/content/reference/_index.md

```diff
@@ -263,18 +263,18 @@ grouped by the originating ARM service.
-| StorageAccountsBlobServicesContainer | 2021-04-01  | v1alpha1api20210401 | v2.0.0-alpha.1 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/blobservice/v1alpha1api/v1alpha1api20210401_storageaccountsblobservicescontainer.yaml) |
-| StorageAccountsManagementPolicy      | 2021-04-01  | v1beta20210401      | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/blobservice/v1beta/v1beta20210401_storageaccountsmanagementpolicy.yaml)                |
-| StorageAccountsManagementPolicy      | 2021-04-01  | v1alpha1api20210401 | v2.0.0-alpha.6 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/blobservice/v1alpha1api/v1alpha1api20210401_storageaccountsmanagementpolicy.yaml)      |
-| StorageAccountsQueueService          | 2021-04-01  | v1beta20210401      | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/queueservice/v1beta/v1beta20210401_storageaccountsqueueservice.yaml)                   |
-| StorageAccountsQueueService          | 2021-04-01  | v1alpha1api20210401 | v2.0.0-alpha.5 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/queueservice/v1alpha1api/v1alpha1api20210401_storageaccountsqueueservice.yaml)         |
-| StorageAccountsQueueServicesQueue    | 2021-04-01  | v1beta20210401      | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/queueservice/v1beta/v1beta20210401_storageaccountsqueueservicesqueue.yaml)             |
-| StorageAccountsQueueServicesQueue    | 2021-04-01  | v1alpha1api20210401 | v2.0.0-alpha.5 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/queueservice/v1alpha1api/v1alpha1api20210401_storageaccountsqueueservicesqueue.yaml)   |
+| Resource                             | ARM Version | CRD Version         | Supported From | Sample                                                                                                                                                                |
+|--------------------------------------|-------------|---------------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StorageAccount                       | 2021-04-01  | v1beta20210401      | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/v1beta/v1beta20210401_storageaccount.yaml)                                 |
+| StorageAccount                       | 2021-04-01  | v1alpha1api20210401 | v2.0.0-alpha.1 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/storage/v1alpha1api/v1alpha1api20210401_storageaccount.yaml)                       |
```

Comment:
This doesn't match the actual samples structure, which has `blobservice` or `queueservice` too. Looks like might need a bit more smarts in the generating tool? Or a bit more consistency in the samples folders?